packages/core/src/PackageContext.ts

```diff
@@ -127,7 +127,7 @@ export class PackageContext implements Context {
       return;
     }
     // tslint:disable-next-line:no-console
-    console.warn(this.getMessage(str, depth));
```

Comment:
```
/**
 * The {@link console.warn()} function is an alias for {@link console.error()}.
 */
warn(message?: any, ...optionalParams: any[]): void;
```

Reply:
```
/**
 * The `console.debug()` function is an alias for {@link console.log()}.
 */
debug(message?: any, ...optionalParams: any[]): void;
```

Reply:
can we add a comment describing the above so it doesn't accidentally return